### DISCUSSION TITLE
New release of G1CP

### DISCUSSION THREAD

**Discussion by pawbuj1981** [AmProsius/gothic-1-community-patch#386] [General]
Hi 

Many of us want to know about future release of G1CP. Is there any possibility to help you guys with new release?

**Comment by AmProsius** [comment]
Any help is appreciated. If you want to start with an easy fix, try one in the "To do" column of v1.3: https://github.com/AmProsius/gothic-1-community-patch/projects/10
The fixes with `compatibility: easy` should be easy to implement.

In this issue our workflow is roughly outlined: https://github.com/AmProsius/gothic-1-community-patch/issues/288

If you are looking for a similar fix that is already implemented and may act as a template, search for the desired issue ID in this project view: https://github.com/AmProsius/gothic-1-community-patch/projects/8

However, once #288 is implemented, most of this process will be automated.
